core-ui/src/components/Predefined/Create/AddonsConfigurations/AddonsConfigurations.js

```diff
@@ -111,24 +118,39 @@ export const AddonsConfigurations = ({
       noValidate
     >
       <div className="fd-form-group">
-        <div className="fd-form-item">
-          <K8sNameInput
+        <FormLabel
+          style={{ display: 'block' }}
```

Comment:
There is a problem with styling in Addons
![image](https://user-images.githubusercontent.com/46759954/162166444-17970270-a954-405c-9aef-38c26b3e617a.png)